frontend/services/api.ts

```diff
@@ -48,7 +46,7 @@ const onResponseSuccess = (response) => {
     return {
       data: {
         // Deserialized data
-        data: response.data ? dataFormatter.deserialize(response.data) : null,
+        data: response.data ? deserialize(response.data)?.data : null,
```

Comment:
I've had a hard time tracking it but I think this new library is flawed or not configured correctly. For example, try to load this page: http://localhost:3000/en/project/pd-organization-test-1-clement-s-project.

You'll see that the page crashes because `project.project_images` is `undefined`. Nevertheless, the API responds correctly with the relationship.

Could you have a look?

Reply:
That's the one difference I found when adding this library. When deserialising and if a relationship doesn't have results, the library will deserialise the empty array into `undefined`/`null`. Unfortunately due to that we always assumed we'd have an array for `involved_project_developers` or `project_images`, even if it was empty. 

I thought I had addressed most circumstances but it looks like I missed too. There was also a hook to deal with extracting a project's contacts which wasn't being used here.